.github/actions/markPullRequestsAsDeployed/markPullRequestsAsDeployed.js

```diff
@@ -30,18 +34,75 @@ function getDeployTableMessage(platformResult) {
     }
 }
 
+/**
+ * Get the [added, removed] pairs for the `🔐 LockCashDeploys 🔐` label on StagingDeployCash
+ *
+ * @return {Promise<Array<[string, string]>>}
+ */
+function getLockCashDeploysTimeline() {
+    return GithubUtils.octokit.paginate(GithubUtils.octokit.issues.listEvents, {
+        owner: GithubUtils.GITHUB_OWNER,
+        repo: GithubUtils.GITHUB_REPOSITORY,
+        issue_number: stagingDeployIssueNumber,
+        per_page: 100,
+    }).then((events) => {
+        let pair = [];
+        const startEndPairs = _.compact(_.map(events, ({event, created_at, label}, index) => {
+            if (event === 'labeled' && label.name === '🔐 LockCashDeploys 🔐') {
+                if (pair.length) {
+                    // flush the pair
+                    pair = [];
+                }
+                pair.push(created_at);
+            } else if (event === 'unlabeled' && label.name === '🔐 LockCashDeploys 🔐') {
+                pair.push(created_at);
+            }
+            if (index === events.length - 1 && pair.length === 1) {
+                pair.push(moment().toISOString());
```

Comment:
Hmmm I'm struggling to understand this ... why are you saving the current moment at the time of this code's execution?

Reply:
So for this case, Let's say the Lock label was added but never removed. So I am assuming that the current time will fill that case and the [added, removed] pair will be completed.

Reply:
Okay, thanks